### ISSUE TITLE
Automatically insert "Specifications" and "Browser compatibility" sections into relevant pages

### ISSUE THREAD

**Issue by bershanskiy** [mdn/yari#6390]
Most pages have the following at the end:
```
## Specifications

{{Specifications}}

## Browser compatibility

{{Compat}}
```
these are trivial includes that can be appended automatically to the end of the document unless there is an explicit section that has the same title but different content.

**Comment by wbamberg** [comment]
See also https://github.com/mdn/yari/pull/5971. As I said there, I'm a bit uncomfortable with things like "the ordering and placement of BCD and spec tables" being hardcoded deep inside some Yari magic.

It would be great eventually to make these macros obsolete but I would prefer there to be a human readable (and human-editable) description of a page format that determines what gets included in the page in what order, and Yari follows that order.

**Comment by teoli2003** [comment]
Yes, I second this. I would prefer Yari to use the new `page-type` YAML entry combined with some kind of template associated with it to enforce this.

What is displayed on a page (sections, sidebars, …) is essentially a _content_ decision. That's why we moved `GroupData.json` from Yari to the mdn/content repo last year (even if this won't be the final way of defining sidebars).

**Comment by caugner** [comment]
Closing as won't fix, as this was already low priority since November 2022, and there is not enough evidence here that there is real need for this feature.
